mri/reconstructors/base.py

```diff
@@ -40,12 +40,15 @@ class ReconstructorBase(object):
         operator and adjoint operator. For wavelets, this can be object of
         class WaveletN or WaveletUD2 from mri.operators .
     prox_op: object that implements proximal operator
-        Defines the proximity operator for the regularization function.
+        Defines the proximity operator for the regularization function H.
         For example, for L1 Norm, the proximity operator is Thresholding
     mu: float or np.ndarray
```

Comment:
I think that `mu` should be removed, if the user didn't provide a `prox_op` than `Identity` will be used, and the result correspond to the least square solution

Reply:
Done in latest commit, resolving